modules/sdk/src/main/scala/org/tessellation/sdk/infrastructure/healthcheck/declaration/PeerDeclarationHealthCheck.scala

```diff
@@ -116,10 +116,21 @@ object PeerDeclarationHealthCheck {
                   }
                 case (NegativeOutcome, round) =>
                   round.getRoundIds.flatMap { roundIds =>
-                    logger.info(s"Outcome for key ${key.show}: negative - removing facilitator | Round ids: ${roundIds.show}") >>
-                      consensusStorage.removeRegisteredPeer(key.id) >>
-                      (consensusStorage.addRemovedFacilitator(key.consensusKey, key.id) >>=
-                        consensusManager.checkForStateUpdateSync(key.consensusKey))
+                    consensusStorage
+                      .getRegisteredPeers(key.consensusKey)
+                      .map(_.contains(key.id))
+                      .ifM(
+                        logger.info(
+                          s"Outcome for key ${key.show}: negative - removing facilitator and removing peer from peer list | Round ids: ${roundIds.show}"
+                        ) >>
+                          consensusStorage.removeRegisteredPeer(key.id) >>
+                          clusterStorage.removePeer(key.id) >>
+                          consensusStorage.addRemovedFacilitator(key.consensusKey, key.id) >>=
+                          consensusManager.checkForStateUpdateSync(key.consensusKey),
+                        logger.info(
+                          s"Outcome for key ${key.show}: negative BUT obsolete. Node not found as registered at given height. Not removing | Round ids: ${roundIds.show}"
```

Comment:
is the uppercase needed here? looks weird tbh :D

Reply:
It's just to stress that nothing will happen. Of course it's not necessary :D

Reply:
imo I'd remove that. we have many places that say nothing is happening. It can be even a debug level btw

Reply:
if we want to emphasize something, imo it should be done by log levels

Reply:
Not a problem, that's just an artists invention.

Reply:
I'm all in favor of not using uppercase nor using exclamation marks in logs.